package/kernel/linux/modules/other.mk

```diff
@@ -954,3 +954,56 @@ define KernelPackage/bmp085-spi/description
 endef
 
 $(eval $(call KernelPackage,bmp085-spi))
+
+define KernelPackage/tpm
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=TPM Hardware Support
+  KCONFIG:= CONFIG_TCG_TPM \
+	CONFIG_TCG_TIS=n \
+	CONFIG_TCG_TIS_I2C_ATMEL=n \
+	CONFIG_TCG_TIS_I2C_INFINEON=n \
+	CONFIG_TCG_TIS_I2C_NUVOTON=n \
+	CONFIG_TCG_ATMEL=n \
+	CONFIG_TCG_TIS_ST33ZP24=n
+  FILES:= $(LINUX_DIR)/drivers/char/tpm/tpm.ko
```

Comment:
can you please add the =n lines into the generic config files please and drop them from this file.